src/main/java/seedu/address/logic/commands/ListItemCommand.java

```diff
@@ -1,25 +1,23 @@
 package seedu.address.logic.commands;
 
 import static java.util.Objects.requireNonNull;
-import static seedu.address.model.Model.PREDICATE_SHOW_ALL_ITEMS;
 
 import seedu.address.model.Model;
+import seedu.address.ui.View;
 
 
 /**
- * Lists all items in the address book to the user.
+ * Lists all items in the inventory to the user.
  */
 public class ListItemCommand extends Command {
 
     public static final String COMMAND_WORD = "listi";
 
```

Comment:
Maybe should have a MESSAGE_USAGE string like if someone does sth like 
listi jkdnkjvsdsn 
or something ( unsure if will return error here)